### ISSUE TITLE
A11y_Azure Tools for VS Code_Azure Function_Refresh_Screenreader: JAWS is not announcing the name of controls.

### ISSUE THREAD

**Issue by kjawale99** [microsoft/vscode-azurefunctions#3122]
**GitHubTags:**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Azure Tools for VS Code; #BM_Azure Tools for VS Code_Win32_Apr22; #Win32; #FTP; #JAWS; #Screenreader; 

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps:**

1. Open Visual studio code
2. Tab till extension button and search for "Azure Functions"
3. Tab till install and press enter now tab till the "Azure" button and press enter.
4. Tab till Refresh button from toolbar.
5. Turn on the JAWS.
6. Check is JAWS is announcing proper name for controls.

**Actual:**
JAWS is not announcing the name of Refresh , Collapse all, Deploy to function app, Select Subscription buttons.

Note: This issue is not observed in NVDA.

**Expected Result:**
JAWS should announce  the name of Refresh , Collapse all, Deploy to function app, Select Subscription buttons.

**Attachment**
![A11y_Azure Tools for VS Code_Azure Function_Refresh_Screenreader](https://user-images.githubusercontent.com/98745784/164401951-541ddd8c-d9be-4b99-9bd9-f25bca4bd5fb.gif)
:

**Comment by nturinski** [comment]
Does this also reproduce with NVDA?  If it does, this is more likely an issue with JAWS than with VS Code's UI.

**Comment by kjawale99** [comment]
@nturinski This issue is not reproducing with NVDA.

**Comment by MicroFish91** [comment]
Hi @kjawale99, I am unable to reproduce the issue you are running into.  See below when I run the same steps locally with JAWS.

<img src="https://user-images.githubusercontent.com/40250218/168185781-c9bb7413-f3d0-464b-844c-6c6b2b3e4481.gif" height="400px">

Same result when I tab from the Azure button as well.

By chance, what version of VS Code and JAWS are you running?

Thank you.

**Comment by alexweininger** [comment]
I think the issue is with this button
<img width="833" alt="Screen Shot 2022-05-12 at 5 14 55 PM" src="https://user-images.githubusercontent.com/12476526/168187316-5594f392-4bd4-4275-9700-edf3b37ce098.png">

**Comment by MicroFish91** [comment]
Hmm, that button is also showing up fine for me when I tab over using JAWS, is it different for any of you?

<img src="https://user-images.githubusercontent.com/40250218/168402695-094ffba6-6b03-4085-876b-0f4341098546.png" height="400px">

**Comment by kjawale99** [comment]
@MicroFish91 I have verifies this issue. The issue is not reproducing. Closing the issue.

![Presentation1](https://user-images.githubusercontent.com/98745784/169463311-2c0743c3-3f59-4fe9-8394-44b24385c6a7.gif)

**Comment by MicroFish91** [comment]
GithubTags:#Closed;
